src/views/jobManager/editJob/systemTab.ts

```diff
@@ -0,0 +1,394 @@
+          text: `Commit (true)`,
+          selected: options["auto commit"] === `true`,
+        },
+        {
+          value: `false`,
+          description: `False`,
+          text: `No commit (false)`,
+          selected: options["auto commit"] === `false`,
+        },
+      ],
+      `Specifies whether auto-commit mode is the default connection mode for new connections. Calling AS400JDBCConnection.setAutoCommit(boolean) will override this property on a per-connection basis. Note that, in order to use transaction isolation levels other than *NONE when using auto-commit mode, the property "true autocommit" needs to be set to true.`
+    )
+    .addSelect(
+      `concurrent access resolution`,
+      `Concurrent access resolution`,
+      [
+        {
+          value: `1`,
+          description: `1`,
+          text: `1`,
+          selected: options["concurrent access resolution"] === `1`,
+        },
+        {
+          value: `2`,
+          description: `2`,
+          text: `2`,
+          selected: options["concurrent access resolution"] === `2`,
+        },
+        {
+          value: `3`,
+          description: `3`,
+          text: `3`,
+          selected: options["concurrent access resolution"] === `3`,
+        },
+      ],
+      `Specifies whether "currently committed" access is used on the connection. A value of 1 indicates that "currently committed" will be used. A value of 2 indicates that "wait for outcome" will be used. A value of 3 indicates that "skip locks" will be used.`
+    )
+    .addSelect(
+      `cursor hold`,
+      `Cursor hold`,
+      [
+        {
+          value: `true`,
+          description: `True`,
+          text: `true`,
+          selected: options["cursor hold"] === `true`,
+        },
+        {
+          value: `false`,
+          description: `False`,
+          text: `false`,
+          selected: options["cursor hold"] === `false`,
+        },
+      ],
+      `	Specifies whether to hold the cursor across transactions. If this property is set to "true", cursors are not closed when a transaction is committed. All resources acquired during the unit of work are held, but locks on specific rows and objects implicitly acquired during the unit of work are released.`
+    )
+    .addSelect(
+      `cursor sensitivity`,
+      `Cursor sensitivity`,
+      [
+        {
+          value: `asensitive`,
+          description: `Asensitive`,
+          text: `asensitive`,
+          selected: options["cursor sensitivity"] === `asensitive`,
+        },
+        {
+          value: `insensitive`,
+          description: `Insensitive`,
+          text: `insensitive`,
+          selected: options["cursor sensitivity"] === `insensitive`,
+        },
+        {
+          value: `sensitive`,
+          description: `Sensitive`,
+          text: `sensitive`,
+          selected: options["cursor sensitivity"] === `sensitive`,
+        },
+      ],
+      formatDescription(autoCommitText)
```

Comment:
Does this belong here?  Looks out of place.

Reply:
Yes, you're right. Cursor sensitivity doesn't belong in this instance.

@ajshedivy Let's remove this one.